tests/aws/aws_kubernetes.rs

```diff
@@ -64,6 +71,82 @@ fn create_and_destroy_eks_cluster(region: &str, secrets: FuncTestsSecrets, test_
             context.clone(),
             generate_cluster_id(region).as_str(),
             generate_cluster_id(region).as_str(),
+            boot_version,
+            region,
+            &aws,
+            &cloudflare,
+            eks_options(secrets.clone()),
+            nodes.clone(),
+        );
+
+        // Deploy
+        if let Err(err) = tx.create_kubernetes(&kubernetes) {
+            panic!("{:?}", err)
```

Comment:
Ouch!!! Don't use `panic!(..)` this is going to crash the application without any recovery. This is super dangerous ☠️. Return an error instead. You will avoir to spend your weekend to troubleshoot why the engine crash :)

Reply:
ha sorry it's tests :) so it's good